packages/origin-247-certificate/src/offchain-certificate/offchain-certificate.service.ts

```diff
@@ -122,4 +154,17 @@ export class OffchainCertificateService<T = null> {
         await this.certEventRepo.save(event, command.id);
         await this.readModelRepo.save(aggregate.getCertificate()!);
     }
+
+    private groupCommandsByCertificate<T extends { certificateId }>(
```

Comment:
why not just `_.groupBy`?

Reply:
There is no lodash in certificate package and I don't want to add it for a single usecase.